control-plane/roles/metal/files/metal-control-plane/templates/metal-console.yaml

```diff
@@ -1,4 +1,17 @@
 ---
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app: bmc-proxy-certs
+  name: bmc-proxy-certs
+data:
+  server_key.pem: {{ .Values.bmc_proxy_certs_server_key_enc }}
```

Comment:
Why does the control plane need the server certs? Doesn't it only connect as a client to the bmc-reverse-proxy instances?